examples/simple-vpc/outputs.tf

```diff
@@ -18,22 +18,22 @@ output "vpc_cidr_block" {
 # Subnets
 output "private_subnets" {
   description = "List of IDs of private subnets"
-  value       = ["${module.vpc.private_subnets}"]
+  value       = [module.vpc.private_subnets]
```

Comment:
The square brackets need to be removed.

Per 0.12 migration guide, the following is no longer necessary.
> Wrapping redundant list brackets ([ and ]) around splat expressions in order to force them to be interpreted as lists even when there are unknown items in the list.